packages/@okta/vuepress-site/docs/concepts/index.md

```diff
@@ -7,8 +7,16 @@ title: Concepts
 
 Understand the basics of Okta's APIs and architecture.
 
-## Featured Concepts
+## Featured concepts
 
 <Cards>
-  <Card href="/docs/concepts/oauth-openid/" cardTitle="OAuth 2.0 Overview">An overview of OAuth 2.0 and OpenID Connect and their Okta implementations. This guide helps you determine what flow is best for the app you are building.</Card>
+  <Card href="/docs/guides/oie-intro/" cardTitle="Okta Identity Engine overview">A high-level introduction to Okta Identity Engine, Okta's new authentication pipeline that provides valuable new features and a more flexible approach to your auth needs.</Card>
+  <Card href="/docs/concepts/oauth-openid/" cardTitle="OAuth 2.0 and OpenID Connect overview">An overview of OAuth 2.0 and OpenID Connect and their Okta implementations. This guide helps you determine what flow is best for the app you are building.</Card>
+  <Card href="/docs/concepts/redirect-vs-embedded/" cardTitle="Okta deployment models — redirect vs. embedded">A look at the different authentication options that Okta provides &mdash; redirect and embedded authentication &mdash; and what the differences are between them.</Card>
 </Cards>
+<br>
+<Cards>
+  <Card href="/docs/concepts/interaction-code/" cardTitle="Interaction Code grant type">An explanation of the Interaction Code grant type, which allows apps to manage user interactions with the authorization server directly.</Card>
```

Comment:
My concern is that, if we're going to use an adverb, its impact is diminished by appearing at the end of the sentence.
```suggestion
  <Card href="/docs/concepts/interaction-code/" cardTitle="Interaction Code grant type">An explanation of the Interaction Code grant type, which allows apps to directly manage user interactions with the authorization server.</Card>
```